x-pack/plugins/security/server/index.ts

```diff
@@ -26,6 +26,9 @@ export {
 } from './authentication';
 export { SecurityPluginSetup };
 export { AuthenticatedUser } from '../common/model';
+// The featurePrivilegeIterator should only be consumed within tests to verify that feature privileges
+// are setup correctly.
+export { featurePrivilegeIterator } from './authorization';
```

Comment:
Instead of exporting it from here, is it possible to have the tests require directly from the `./authorization` module?

Reply:
I think we _can_, but it's my understanding that we should export anything we consider part of the plugin's public contract from the `index.ts` files in the NP. Since this is consumed by other plugins, I decided to put it here. @azasypkin do you agree, or am I mis-understanding?

FWIW, the linter doesn't seem to complain when I do this, so _maybe_ this is OK to do?

![image](https://user-images.githubusercontent.com/3493255/75355339-5694c200-587c-11ea-9517-a95ec1d3dd78.png)

Reply:
Another thought -- I had the `featurePrivilegeIterator` as part of the security plugin, because it has to have an understanding of the privilege model to iterate and merge sub feature privileges. We could instead move this iterator to the `features` plugin though, and import it into the security plugin, and elsewhere as needed.

Reply:
**update**: moved this to the `./authorization` module as @kobelb suggested, since the linter doesn't complain about this. Easy enough to change if we don't like it!

Reply:
I like the change, but that's likely just because I suggested it :) I think imports like this are "fine" if they're just for tests, but I guess time will tell!